### ISSUE TITLE
bottom padding or extra space

### ISSUE THREAD

**Issue by bsodi** [maykar/custom-header#72]
hi,
please please help looked everywhere and been bugging me for the pass week now.  using custom header and i was able to do the top padding adjustment but now since updated to 103.5 the bottom is the issue.  i tried adjusting the bottom and it does make it bigger but not smaller.  Red is the background and using picture elements.  please please help.  using raspberry chrome kiosk.  screen is 1280,800 with touchscreen.  the red bottom shows on desktop and on tablets.  
Installed: 1.1.7
```
custom_header:
  compact_mode: true
  disabled_mode: false
  hide_help: true
  footer_mode: true
  exceptions:
    - conditions:
        user: Kiosk1
      config:
        default_tab: home_1280
        hide_tabs: 'Home_1200, home,'
        panel_view_css: 'padding-top:96px; padding-left:0px; padding-right:0px; padding-bottom:0px;'
        kiosk_mode: true
```
![screenshot-bottom over](https://user-images.githubusercontent.com/58150197/71624180-454a7480-2baf-11ea-958a-288e85fe1274.png)

**Comment by maykar** [comment]
Can you try `panel_view_css: 'padding-top:96px; padding-left:0px; padding-right:0px; padding-bottom:0px; margin-bottom: -96px'`

**Comment by maykar** [comment]
Feel free to keep this thread going, but for now I'm closing as stale

**Comment by bsodi** [comment]
hi Maykar,
thanks for the reply.  Sorry was away on business trip.  Did not fix above config.  it only happens when the header is hidden/kiosk mode.  if you enable the header the extra space is not there.

**Comment by maykar** [comment]
Thanks, that's helpful. I think I see where the issue lies, reopening.

**Comment by maykar** [comment]
Would you mind sharing your lovelace config for that view?

**Comment by bsodi** [comment]
[kiosk1280_800_main_view.txt](https://github.com/maykar/custom-header/files/4048740/kiosk1280_800_main_view.txt)

sorry im still new at this... i learn as i go... i dont know where to upload the entire view with background and so...i've added the txt format of the lovelace view

**Comment by xannor** [comment]
I want to add that I see this as well. I have a simple tab with a weather card and the tab is set to panel. When kiosk mode is enabled it will cause a scroll bar and have a small background color gap at the bottom. This does not happen when panel is not enabled for the tab, or when the header is visible but the sidebar is hidden. (I tested this with just disable header, then just disable sidebar.)

**Comment by bsodi** [comment]
correct only happens during the kiosk mode.  The annoying part is the scroll bar on my wall panel running fullscreen chromium on rasp3.  thanks again for helping and hopefully you can fix :).  If you know way to disable the top bar without addon please share

**Comment by maykar** [comment]
Please, try the latest release 1.2.1 and report back if the issue persists or close the issue if it has been solved.

**Comment by bsodi** [comment]
sorry no change... tried above config as well

**Comment by maykar** [comment]
Can you remove any `panel_view_css` and confirm this isn't a cache issue by checking the version number in your browsers dev tools?

**Comment by bsodi** [comment]
i cleared the cache and remove all panel_view_css.  this time the top padding is correct no need to adjust.  but the bottom still same with scroll bar on the side due to extra spacing.  
if theres a way to swap the css header with this https://community.home-assistant.io/t/custom-header-template-showcase-and-discussion/158166/7
i'll be happy.  i added the custom header_text but if there is a way to get rid of the tabs and put this scrolling text in it place would be even better.

**Comment by maykar** [comment]
`"if there is a way to get rid of the tabs"`

You can use `tab_container_css: 'display: none;'` to hide the tab container, but you're going to have to do some modification of `view_css` and `panel_view_css`

But, I'd like to nail down what your issue is with panel view. Is the correct version reported in your dev tools console in your browser?

**Comment by maykar** [comment]
Sorry, that might be difficult to do on kiosk chromium. Might also be worth double checking or reinstalling 1.2.1 in hacs.

**Comment by bsodi** [comment]
Great work my friend.  Deleting and reinstalling did it.  So uninstalled, restarted HA... installed Restarted HA. thanks again..

**Comment by bsodi** [comment]
attached pic...
![fixed bottom header](https://user-images.githubusercontent.com/58150197/72307809-93e51d80-3649-11ea-8046-83476b6fefc8.jpg)

**Comment by maykar** [comment]
I know that is probably unfinished, but I kind of dig the industrial look with that interface

**Comment by bsodi** [comment]
Lol its my test unit in the basement where I spend most of my free time during winter months.  Upstairs in the kitchen I have the Samsung 8” tablet.  I do electronic but wished I learned CSS and java.  I’m can copy and past codes but would love to learn the backend.  This weekend I made my fireplace smart using Sonoff SV, temp sensor and light sensor. I will soon add the fireplace UI to this front end. Is there way to make my weather summery scroll like the link I forwarded above inside the card?

![87E3F982-D451-4A6E-8E75-031359238D37](https://user-images.githubusercontent.com/58150197/72310512-f214ff00-364f-11ea-96b5-3b1816fcb710.jpeg)
![6412DFBB-1F12-4401-9FFF-4907C56BE19D](https://user-images.githubusercontent.com/58150197/72310513-f214ff00-364f-11ea-9e8e-1624a032b189.jpeg)
